services/gafaelfawr/values-nublado.yaml

```diff
@@ -0,0 +1,55 @@
+gafaelfawr:
+  # Do not specify ingress.host because we're using the wildcard virtual host.
+  host: nublado.lsst.codes
+
+  vault_secrets:
+    path: "secret/k8s_operator/nublado.lsst.codes/jwt_authorizer"
+  
+  # Session length/Token expiration (in minutes) (30 days)
+  session_length: 43200
+  
+  # Existing PVC for redis claim
+  # If empty, redis will use emptydir
+  redis_claim: ""
+  
+  # <client id from cilogon.org or OIDC provider>
+  oauth2_proxy_client_id: "cilogon:/client_id/12fec51db55c36b208dfbe2dee78065c"
+  
+  # Default capability required for user-based operations (like generating a token)
+  user_capability: "exec:user"
+  
+  # Use the PR branch that supports GitHub.
+  image: "lsstdm/jwt_authorizer:tickets-DM-23776"
+
+  config:
+    loglevel: DEBUG
+  
+    github:
+      client_id: a19e79298a352f3e5650
+
+    # Defining all capabilites applications may need.
+    # This feeds into the token downlad page
+    known_capabilities:
+      exec:portal: Use the Portal to execute operations. Mostly used from notebook APIs.
+      read:image: Read images from the SODA and other image retrieval interfaces
+      read:image/metadata: Read image metadata from SIA and other image interfaces
+      read:tap: Execute SELECT queries in the TAP interface on project datasets
+      read:tap/efd: Execute SELECT queries in the TAP interface on EFD datasets
+      read:tap/user: Execute SELECT queries in the TAP interface on your data
+      write:tap/user: Upload tables to your database workspace
+      read:tap/history: Read the history of your TAP queries.
+      read:workspace: Read project datasets from the file workspace
+      read:workspace/user: Read the data in your file workspace
+      write:workspace/user: Write data to your file workspace
+  
+    # Mapping of capabilities to groups that are part of the `isMemberOf` claim
+    group_mapping:
```

Comment:
are these the same as in the values.yaml?  if so I'd probably just leave it (same for capabilities) - otherwise I'd assume there's something different and start diffing to find it

Reply:
Oh, yes, good point. I've made this file much smaller.